go/cmd/ctrl/main.go

```diff
@@ -54,6 +54,8 @@ var Cmd = &cli.Command{
 		// Control Plane Specific
 		cli.String("auth-token", "Authentication token for control plane API access. Required for secure deployments.",
 			cli.EnvVar("UNKEY_AUTH_TOKEN")),
+		cli.String("api-key", "API key for simple authentication (demo purposes only). Will be replaced with JWT authentication.",
+			cli.EnvVar("UNKEY_API_KEY")),
```

Comment:
this should be required, there's a util like `cli.Required()` which you can add in the arguments

Reply:
done